osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/ws/OrganisationIdentificationClientInterceptor.java

```diff
@@ -92,6 +95,11 @@ public static Builder newBuilder() {
 
     @Override
     public boolean handleRequest(final MessageContext messageContext) {
+        // TODO (RvM): remove logging:
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)